packages/synapse-react-client/src/lib/containers/auth/TwoFactorEnrollmentForm.tsx

```diff
@@ -0,0 +1,259 @@
+            {false && 'or NIH RAS'} account, you may need to use 2FA as part of
+            those processes as well.
+            {TWO_FACTOR_DOCS_LINK.length > 0 && (
+              //   Remove conditionality when docs are ready
+              <>
+                {' '}
+                <Link href={TWO_FACTOR_DOCS_LINK}>Learn more about 2FA</Link>.
+              </>
+            )}
+          </Typography>
+        </Section>
+        <Section>
+          <SectionInnerGrid>
+            <Box>
+              <Typography variant="body1" sx={{ fontWeight: 700 }}>
+                Step 1. Get one of these apps
+              </Typography>
+              <Typography variant="body1">
+                Download and install one of these authentication apps to set up
+                2FA on your mobile device, tablet, or desktop. If you have one
+                of these apps already installed, you can skip this step.
+              </Typography>
+            </Box>
+            <Stack justifyContent={'space-between'} sx={{ flexShrink: 0 }}>
+              <Link href={'https://authy.com/download/'}>Authy</Link>
+              <Link
+                href={
+                  'https://duo.com/product/multi-factor-authentication-mfa/duo-mobile-app'
+                }
+              >
+                DUO Mobile
+              </Link>
+              <Link href={'https://googleauthenticator.net/'}>
+                Google Authenticator
+              </Link>
+              <Link
+                href={
+                  'https://www.microsoft.com/en-us/security/mobile-authenticator-app'
+                }
+              >
+                Microsoft Authenticator
+              </Link>
+            </Stack>
+          </SectionInnerGrid>
+        </Section>
+        <Divider />
+        <Section>
+          <SectionInnerGrid>
+            <Box>
+              <Typography variant="body1" sx={{ fontWeight: 700 }}>
+                Step 2. Scan the QR Code
+              </Typography>
+              <Typography component="ol" variant="body1" sx={{ pl: 2, my: 1 }}>
+                <li>Open your authenticator app</li>
+                <li>Tap the &quot;+&quot; button</li>
+                <li>Use your camera to scan the image to the right</li>
+              </Typography>
+              <Link
+                onClick={() => {
+                  setShowSecretInModal(true)
+                }}
+              >
+                Can&apos;t use your camera?
+              </Link>
+              <TwoFactorSecretDialog
+                secret={totpSecret?.secret ?? ''}
+                open={showSecretInModal}
+                onHide={() => setShowSecretInModal(false)}
+              />
+            </Box>
+            <Box
+              display={'flex'}
+              justifyContent={'center'}
+              alignItems={'middle'}
+              height={'auto'}
+            >
+              {!hasQrCode && <SynapseSpinner size={50} />}
```

Comment:
Later, we can use theme/context to use different branded spinner, or just replace this with unbranded